api/hearings/models/hearingActuals.model.ts

```diff
@@ -0,0 +1,89 @@
+import { HMCStatus } from './hearings.enum';
+
+export interface HearingActualsModel {
+  hearingActuals: HearingActuals;
+  hearingPlanned: HearingPlanned;
+  hmcStatus: HMCStatus;
+}
+
+export interface HearingPlanned {
+  plannedHearingType: number;
+  plannedHearingDays: PlannedHearing[];
+}
+
+export interface PlannedHearing {
+  plannedStartTime: string;
+  plannedEndTime: string;
+  parties: Party[];
+}
+
+export interface Party {
+  partyId: number;
+  partyRole: number;
+  individualDetails: IndividualDetail;
+  organisationDetails: OrganisationDetails;
+  partyChannelSubType: string;
+}
+
+interface IndividualDetail {
+  title: string;
+  firstName: string;
+  lastName: string;
+}
+
+interface OrganisationDetails {
+  name: string;
+  cftOrganisationID: string;
+}
+
+export interface HearingActuals {
+  hearingOutcome: HearingOutcome;
+  actualHearingDays: ActualHearingDay[];
+}
+
+interface HearingOutcome {
+  hearingFinalFlag: boolean;
+  hearingResult: HearingResult;
+  hearingResultDate: string;
+  hearingResultReasonType: string;
+  hearingType: string;
+}
+
+export interface ActualHearingDay {
+  hearingDate: string;
+  hearingStartTime: string;
+  hearingEndTime: string;
+  pauseDateTimes: PauseDateTime[];
+  actualDayParties: ActualDayParty[];
+}
+
+export enum HearingResult {
+  COMPLETED = 'COMPLETED',
+  ADJOURNED = 'ADJOURNED',
+  CANCELLED = 'CANCELLED',
+}
```

Comment:
could we move all those enums to hearings.enum.ts? The benefit is when we create enum, we need to check whether it's existing.